setup.sh

```
@@ -26,7 +26,7 @@ create_link() {
 
     # 0. リンク元ファイルの存在確認
     if [ ! -f "$src" ]; then
-        echo "ERROR: Source file not found: $src"
+        printf "ERROR: Source file not found: $src\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)